### PR TITLE
update #5083 uvnc.com

### PR DESCRIPTION
**Pull request by wout-codes**
#5083

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/5083
+uvnc.com#?#.sp-module:contains(adsbygoogle)
```
Comment (Alex-302) [inline EnglishFilter/sections/css_extended.txt:5]: why not `:has()`? `:contains()` is slower.

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -23,12 +23,12 @@
...
...xgen.com|desktophut.com|cayota.hulnews.top|infofuge.com|moviedhun.xyz|pinsystem.co.uk|furioscans.com|mastiflix.com|grafixfather.com|streamcr7.com|globalrph.com|phoenixfansub.com|cyberdefensemagazine.com|vectogravic.com|teknotaci.com|phuquoc.me|uplod.net|onlinecoursebay.com|sshuebner.org|lectulandia.me|scrapywar.com|worldpremiumware.online|godmods.com|modagamers.com|tipslearn.com|nulljungle.com|downloadcursos.net|hitsebeats.eu|fikiri.net|merkulov.design|epidemia-koronawirus.pl|cardiagn.com|nulleb.com|javgayplus.com|dr-farfar.com|netfuck.net|allywebsite.com|temi.pl|otakupdf.com|xenweb.net|darmowatv.ws|sonradangamer.com|onlinefreecourse.net|themeslide.com|phpscripttr.com|tutomoviles.com|frkmusic.me|freewebcart.com|choq.fm|zone-annuaire.website|exdb.net|pentruea.com|software-on.com|restplay.xyz|zone-annuaire.xyz|tunovelaligera.com|frkmusic.cc|open3dmodel.com|nullpk.com|baixarsoftware.com|dr-farfar.com|fcportables.com|webphanmem.com|nulledpremium.com|nsfw247.to|boycracked.co|elrellano.com|masihbelajar.com|tehnografi.com|myqqjd.com|filecr.com|idolprojects.com|usb-antivirus.com|jprealtime.com|kursors.lv|novelmultiverse.com|bollyholic.site|mundowuxia.com|adibenha.com|worldfree4u-lol.online
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt:26]: jpvariety.com - if you delete unnecessary domain not related to the issue(dead, or the rule is not required) better add to comment why removed.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt]: Bad removal. I am clumsy.

**Review by Alex-302 — APPROVED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/css_extended.txt:5]: Will change to `uvnc.com#?##sp-left > div.sp-column > div.sp-module:has(.adsbygoogle)`

**Pull request merged**